src/app/modules/consultations/consultation-profile/consultation-profile.graphql.ts

```diff
@@ -31,10 +31,36 @@ export const ConsultationProfile = gql`
       reviewType
       satisfactionRatingDistribution
       status
+      anonymousResponses {
```

Comment:
@Nishanthh Don't need to display anonymousResponses in the consultation-profile page. Please remove

Reply:
Only use this to display the current user's response